tracpro/settings/deploy.py

```diff
@@ -72,6 +72,12 @@
 
 STATIC_ROOT = os.path.join(WEBSERVER_ROOT, 'public', 'static')
 
+if from_env('CUSTOM_STATICFILES_DIR'):
+    STATICFILES_DIRS.insert(0, os.path.join(WEBSERVER_ROOT, from_env('CUSTOM_STATICFILES')))
```

Comment:
I think the environment variables should contain the full path rather than a directory relative to `WEBSERVER_ROOT` which isn't configurable.